src/test/kotlin/fi/metatavu/oss/api/test/functional/tests/DeviceSurveysTestIT.kt

```diff
@@ -445,4 +443,63 @@ class DeviceSurveysTestIT: AbstractResourceTest() {
             )
         }
     }
+
+    @Test
+    fun testDeviceSurveysRealtimeNotification() {
+        createTestBuilder().use { testBuilder ->
```

Comment:
Tests with test builder can also be simplified by using the single-expression syntax.

Reply:
Resolved